portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -88,14 +88,74 @@ definition {
 	}
 
 	@description = "Verify that a Form can be submitted from the Forms Widget"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeSubmittedFromWidget {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131019 CanBeSubmittedFromWidget pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
+
+		var textFieldName = FormFields.getFieldName();
+
+		Form.save();
+
+		Form.publishForm();
+
+		SitePages.addPublicPage(
+			pageName = "Test Page",
+			sitePageType = "Widget Page");
+
+		Navigator.gotoPage(pageName = "Test Page");
+
+		Portlet.addPG(
+			pageName = "Test Page",
+			portletName = "Form");
+
+		FormPortletNavigator.gotoSelectForm();
+
+		FormPortletConfiguration.selectForm(formName = "Form 1");
+
+		Form.save();
```

Comment:
Hi @evanilson-santana, can you modify these lines of code to follow the same pattern as [this](https://github.com/carolmariaabb/liferay-portal/blob/05d05b93a9282ada19e000cd9afde9c9e600824e/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L608-L624) 
one?
For example:
-  `Form Page` instead of `Test Page`
- `PortletEntry.save();` instead of `Form.save();`
- Include the 
```
ProductMenu.gotoPortlet(
	category = "Site Builder",
	portlet = "Pages");
```
before the `SitePages.addPublicPage`